crates/standalone/src/subcommands/start.rs

```diff
@@ -88,15 +123,21 @@ pub fn cli(is_standalone: bool) -> clap::Command {
                 .help("Enable Tracy profiling (SPACETIMEDB_TRACY)"),
         )
         .arg(jwt_pub_key_path_arg)
-        .arg(jwt_priv_key_path_arg)
-        // We still want to keep the executable name `spacetimedb` when we're executing as a standalone, but
-        // we want the executable name to be `spacetime` when we're executing this from the CLI. We have to
-        // pass these strings with static lifetimes so we can't do any dynamic string manipulation here.
-        .after_help(if is_standalone {
-            "Run `spacetimedb help start` for more detailed information."
-        } else {
-            "Run `spacetime help start` for more information."
-        })
+        .arg(jwt_priv_key_path_arg);
+
+    // Standalone databases have the option to run entirely in memory.
```

Comment:
@kulakowski  Why is this standalone only? Can we not also add this functionality to the `spacetime` command as well?

Reply:
Fixed by https://github.com/clockworklabs/SpacetimeDB/pull/148